compose/docker-compose.yml

```diff
@@ -153,19 +166,72 @@ services:
       - ../src/python:/home/broker/python
     working_dir: "/home/broker/python"
     entrypoint: [ "python", "-m", "delivery.UbidotsWriter" ]
+    ports:
+      - "8006:8006" # Expose the metrics port
 
   frred:
     image: broker/python-base
     restart: unless-stopped
     env_file:
       - .env
+    environment:
+      - PYTHONPATH=/home/broker/broker/src/python:/home/broker/broker/test/python
```

Comment:
Is there a reason this environment variable is needed for `frred` now? If so, should it be defined in the `.env` file instead?

Reply:
Agreed, resolved.